docs/eigenlayer/restaking-guides/1-restaking-developer-guide.md

```diff
@@ -215,12 +220,12 @@ Withdraw yield shares amount:
 3. Wait for the Escrow Period to complete.
 4. Invoke `DelegationManager.completeQueuedWithdrawal()`. This function can only be invoked by the **EigenPod Owner wallet**.
 
-
+## FAQ
 
 ### Queue withdrawal takes an `amount` as input, what will that value be?
 
 The input amount for `DelegationManager.queueWithdrawal()` can be any amount you like. However, it must be less than or equal to `withdrawableRestakedExecutionLayerGwei`. The value of `withdrawableRestakedExecutionLayerGwei` will equal any checkpointed yield (consensus rewards, fees, ETH) and any checkpointed exited validator native eth that has been withdrawn to the EigenPod.
 
 ### How to account for the exchange rates between Strategy token `amounts` and `shares`?
 
-Invoke `[Strategy].underlyingToSharesView()` and `[Strategy].sharesToUnderlyingView()` as needed to convert their current balances between shares and tokens.
+Invoke `[Strategy].underlyingToSharesView()` and `[Strategy].sharesToUnderlyingView()` as needed to convert their current balances between shares and underlying token amounts.
```

Comment:
You can use `underlyingToShares` and `sharesToUnderlying` -- the "View" tacked on to the end of both methods is not needed.

Reply:
Awesome, thanks. Updated functions to remove View suffix.